docs/supportedchains.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: Supported Chains
+sidebar_position: 9
+---
+
+# Supported Chains for Biconomy SDK
+
+
+## Mainnet Chains
+1. Ethereum
+2. Polygon Mainnet
+3. BSC Mainnet
+4. Polygon zkEVM Mainnet
+5. Arbitrum One Mainnet
+6. Arbitrum Nove Mainnet
+7. Optimism Mainnet
+8. Avalanche Mainnet
+9. Moonbeam Mainnet
```

Comment:
remove Moonbeam since its not supported on across all services

Reply:
noted.